Applications/Tizen_native/CustomShortcut/README.md

```diff
@@ -16,14 +16,31 @@ You can use `tizen studio gui` to build and run.
 
 If you want to do in CLI, you first need to [convert the project to CLI](https://developer.tizen.org/ko/development/tizen-studio/native-tools/cli/converting-projects-cli) and do as follows.
 
-for the time being, you have to install and prepare nntrainer, capi-nntrainer to run this program
 
 ### Prerequisite
 
 - [NNTrainer Prerequisite](https://github.com/nnstreamer/nntrainer#prerequisites)
 - tizen studio 3.7
 - wearable 6.0 rootstrap (built after Aug 25 2020)
 - appropriate wearable tizen device with tizen 6.0 installed (built after Aug 25 2020)
+- gstreamer plugin png image decoder (https://gstreamer.freedesktop.org/documentation/png/pngdec.html?gi-language=c)
+
+### Install `pngdec`
+
+Currently, `pngdec` gstreamer element is required but it is not installed in the wearable packaging.
```

Comment:
it is not installed in the wearable packaging -> it is not packaged by default in the released wearable OS.

Reply:
Done. Thanks